lib/atat-web-api-stack.ts

```diff
@@ -42,5 +56,25 @@ export class AtatWebApiStack extends cdk.Stack {
         })
       )
     );
+
+    // State Machine
```

Comment:
@kylelaker-ccpo were you envisioning seeing all of this functionality wrapped into a separate Construct as a means to modularize the Stack as much as possible?